src/Sentry/Internal/MainSentryEventProcessor.cs

```diff
@@ -56,6 +57,7 @@ public SentryEvent Process(SentryEvent @event)
             }
 
             AddMemoryInfo(@event.Contexts);
+            AddThreadPoolInfo(@event.Contexts);
```

Comment:
@bruno-garcia  we only want that information for Events?

Reply:
forgive my ignorance, where else would we have it?

Reply:
I believe Lucas is referring to Transactions as the other possible place to add it.
We only run `SentryEvent` instances here so transactions won't be getting this information.

A few things go through when deciding: 
* Does this information help understand performance problems with the app?
* Is it cheap to retrieve this information (as in resource intensiveness)?

If the answer is 'yes' to both, it's a good candidate. Ideally we add information that's relative to the transaction itself (like what happened during the duration of that transaction). Like: How many GC gen0/1/2 ran during the transaction, for example. As 'point in time' information alone within a transaction might not be as valuable. 

Ultimately, this might be more useful only as Metrics. Which is something we'll look into adding in the future